frontend/src/components/Cards/ForYouHome.tsx

```diff
@@ -0,0 +1,34 @@
+import { Link } from 'react-router-dom'
+import CardReport from '~/components/Cards/CardReport'
+import { cardForYou } from '~/data/fakeData'
```

Comment:
Sửa tên lại cho chuẩn
```suggestion
import { dataForYou } from '~/data/fakeData'
```